CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 0.13.0
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/watchmaker/compare/0.12.1...0.13.0)
+
+**Released**: 2018.01.29
+
+**Summary**:
+
+*   amazon-inspector-formula 
+    - New salt formula distributed with watchmaker
+    - installs amazon-inspector formula
```

Comment:
Also need a line item for `ash-linux-formula` that summarizes the changes there. See its commit log.

Reply:
Done